independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -35,6 +35,14 @@
  */
 public class QuarkusClassLoader extends ClassLoader implements Closeable {
     private static final Logger log = Logger.getLogger(QuarkusClassLoader.class);
+    private static final Logger lifecycleLog = Logger.getLogger(QuarkusClassLoader.class.getName() + ".lifecycle");
+    private static final boolean LOG_ACCESS_TO_CLOSED_CLASS_LOADERS = Boolean
+            .getBoolean("quarkus-log-access-to-closed-class-loaders");
+
+    private static final short STATUS_OPEN = 1;
+    private static final short STATUS_CLOSING = 0;
+    private static final short STATUS_CLOSED = -1;
```

Comment:
Minor nitpick: in most places I've seen, these are referred to as states, not statuses. Obviously it makes very little difference at the end of the day :)

Reply:
We already have something called `state` in this class so I went with what's left :).

Reply:
Now I could name it `open`?

Reply:
Sure yeah

Reply:
In the end, I will keep `status` as I ended up with things like `open = CLOSING` or `open < OPEN` and this is not really ideal.

If at some point we decide to rename `state`, we could adjust but for now, that's how it will be.